docs/selectively-triggering-aktualizr.adoc

```diff
@@ -58,3 +58,20 @@ Installation reports are sent when aktualizr polls the server for updates, so th
 
     aktualizr --running-mode=check
 
+=== Check for campaigns target the device
+
+In the OTA Connect web app, you can create smart groups that filter for devices based on specific criteria. When OTA Connect detects a new device that matches this criteria, it automatically publishes the campaign to the device. This running mode checks to see if the device matches the filter criteria used in an active campaign.
```

Comment:
Evoking smart groups seems a bit unnecessary. An user can create campaigns with static groups as well and I'd say it's a detail of the campaign creation process.

Maybe just a brief summary of what a campaign is in abstract (setup of an update to a fleet, with associated metadata) would suffice?

Reply:
Ok added a more general description.